### PR TITLE
Make relocate command works immediately

### PR DESCRIPTION
**Pull request by LIUJIAHAOCS**

### CODE REVIEW

**Review by deltango — APPROVED** [review]
Good job with feature enhancement.

File: src/main/java/utask/storage/XmlUTaskStorage.java [hunk 1]
```diff
@@ -72,4 +72,14 @@ public void saveUTask(ReadOnlyUTask uTask, String filePath) throws IOException {
         XmlFileStorage.saveDataToFile(file, new XmlSerializableUTask(uTask));
     }
 
+    @Override
+    public void moveUTask(String prePath, String newPath) {
```
Comment (yong24s) [inline src/main/java/utask/storage/XmlUTaskStorage.java]: Empty implementation. 

May want to assert false

File: src/main/java/utask/staging/ui/StageDriver.java [hunk 2]
```diff
@@ -190,10 +190,13 @@ public void handleExitAppRequestEvent(ExitAppRequestEvent event) {
...
     private void handleFileRelocateEvent(FileRelocateEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
+        String prePath = config.getUTaskFilePath();
         config.setUTaskFilePath(event.getPath() + "/utask.xml");
+        storage.setFilePath(event.getPath() + "/utask.xml");
         //Update config file in case it was missing to begin with or there are new/unused fields
         try {
-            ConfigUtil.saveConfig(config, Config.DEFAULT_CONFIG_FILE);
+            ConfigUtil.saveConfig(config, ConfigUtil.getConfigPath());
+            storage.moveUTask(prePath, event.getPath() + "/utask.xml");
```
Comment (yong24s) [inline src/main/java/utask/staging/ui/StageDriver.java]: Magic String

File: src/main/java/utask/staging/ui/StageDriver.java [hunk 3]
```diff
@@ -190,10 +190,13 @@ public void handleExitAppRequestEvent(ExitAppRequestEvent event) {
     @Subscribe
     private void handleFileRelocateEvent(FileRelocateEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
+        String prePath = config.getUTaskFilePath();
         config.setUTaskFilePath(event.getPath() + "/utask.xml");
+        storage.setFilePath(event.getPath() + "/utask.xml");
```
Comment (yong24s) [inline src/main/java/utask/staging/ui/StageDriver.java]: Magic String

File: src/main/java/utask/commons/core/Config.java [hunk 4]
```diff
@@ -9,6 +9,7 @@
 public class Config {
 
     public static final String DEFAULT_CONFIG_FILE = "config.json";
+    public static String relocatedFile = "";
```
Comment (yong24s) [inline src/main/java/utask/commons/core/Config.java]: No other way to do this?

It's a code smell  - Global variable

**Review by yong24s — CHANGES_REQUESTED** [review]
Consider ways to remove global variable

File: src/main/java/utask/storage/XmlUTaskStorage.java (hunk 1 above)
Comment (yong24s) [reply src/main/java/utask/storage/XmlUTaskStorage.java]: Change to a move general name?

Move saveFile

**Review by yong24s — APPROVED** [review]

**Review by yong24s — APPROVED** [review]
Good work

**Pull request merged**